pandas/core/arrays/datetimes.py

```diff
@@ -1065,19 +1065,19 @@ def date(self):
 
         return tslib.ints_to_pydatetime(timestamps, box="date")
 
-    year = _field_accessor('year', 'Y', "\n The year of the datetime\n")
+    year = _field_accessor('year', 'Y', "The year of the datetime.")
```

Comment:
Same question here - does removing these not have any impact?

Reply:
These were suggested by @datapythonista according to PR #24099. All ‘\n’ should be removed. I am not sure if these ones are supposed to be left there?

Reply:
OK but then this would be in conflict with the above change, so should revert back there

Reply:
I can revert back this one, what about the one above “\n{}\n”?

Reply:
IIUC you want to revert the `\n{}\n` as that's what was implemented with the PR you just linked to. You can then leave these as is